definitions/ext-jfrog-servers/definition.yml

```diff
@@ -0,0 +1,21 @@
+# Derived from the JFROG log ingestion, denotes the server where 
+domain: EXT
+type: JFROG_SERVER
+
+synthesis:
+  identifier: hostname
+  name: hostname  
+  encodeIdentifierInGUID: false
```

Comment:
Is there a particular reason to set this to false ?

Not encoding an identifier is not encouraged as the hostname must not exceed a certain [length](https://github.com/newrelic/entity-definitions/blob/main/docs/guid_spec.md) in this case.

Setting this to true will hash the identifier and is therefore guaranteed to stay within the length boundaries.

Reply:
I was just using another example as a template and this ported over, no real reason to have this as false. I changed it back.